_docs/platforms/kubernetes/install/1.10.md

```diff
@@ -18,10 +18,10 @@ cd storageos/k8s/deploy-storageos/standard
 
 or using the Helm chart:
 ```bash
-git clone https://github.com/storageos/helm-chart.git storageos
-cd storageos
+helm repo add storageos https://charts.storageos.com
+helm repo update
 # Set cluster.join to hostnames or ip addresses of at least one node
-helm install . --name my-release --set cluster.join=node01,node02,node03
+helm install storageos/storageos --name my-release --namespace storageos --set cluster.join=node01,node02,node03
```

Comment:
I think all these helm commands should start with `$`, as the following commands have `$`.
I know those are notes from the helm chart, but it looks weird to have $ at some places and not at others.

Reply:
Every command that can be copy and paste from the docs shouldn't have a $ at the beginning to make easier for the user to run the commands.